cli/flox-rust-sdk/src/models/lockfile.rs

```diff
@@ -3555,7 +3564,7 @@ pub(crate) mod tests {
             Lockfile::check_packages_are_allowed(&vec![foo_locked], &Allows {
                 unfree: None,
                 broken: None,
-                licenses: vec![]
+                licenses: None
```

Comment:
It's worth having one new test case for the new variant we've introduced of `Some(vec![])`.

Reply:
https://github.com/flox/flox/pull/2885